source/dotnet/Library/AdaptiveCards/Rendering/AdaptiveElementRenderers.cs

```diff
@@ -1,16 +1,54 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Reflection;
 using System.Text;
 using System.Threading.Tasks;
 
 namespace AdaptiveCards.Rendering
 {
+    public class AdaptiveActionHandlers
```

Comment:
>AdaptiveActionHandlers [](start = 17, length = 22)

Does this deserve to be in its own file, rather than including this class in another class' file?

Reply:
Def